### PR TITLE
Add release notes for fix

### PR DESCRIPTION
**Pull request by reinsch82**
- #19340

### CODE REVIEW

**Comment by hythloda** [comment]
I thought this section is automated. Your version should be higher up. Also make sure you have more context as to why this was something we wanted to fix and how to use it.

**Review by ljacomet — CHANGES_REQUESTED** [review]
Fixed issues are populated automatically.

Your changes should go above that as Amanda indicated.

**Comment by donat** [comment]
This is not just a bug fix but also a change in behavior. I think it deserves a section in the release notes, especially given that we struggle to add content there.

File: subprojects/docs/src/docs/release/notes.md [hunk 1]
```diff
@@ -86,6 +86,15 @@ TODO - Java lambdas are supported, and unsupported captured values are reported.
 TODO - File collections queried at configuration time are treated as configuration inputs.
 TODO - File system repositories are fully supported including dynamic versions in Maven, Maven local, and Ivy repositories
 
+## Tooling API improvements
+
+- [Build launched via TAPI ignores logger level settings of target build](https://github.com/gradle/gradle/issues/19340)
+
+  `org.gradle.logging.loglevel` settings in `gradle.properties` are now recognized when a build is executed with the Tooling API.
+  This was a problem caused executions triggered from within the IDE to ignore loglevel settings in `gradle.properties`. 
+  A workaround had to be implemented. This workaround can be avoid from 8.1 onwards.
```
Comment (hythloda) [inline subprojects/docs/src/docs/release/notes.md:95]: Here is the recommended format for each section: 
- Set the context - what are we even talking about (don’t assume the user already knows)
- Express the problem or use case in the given context
- Explain how the new release addresses that problem or use case
- Provide build script snippet or screenshot illustrating the new feature, if applicable
- Link to the full documentation for more details

Comment (reinsch82) [reply subprojects/docs/src/docs/release/notes.md]: I hope this comes closer to the recommendation now.
I think a screenshot is not helpful here, at least I can't think of one.

**Review by hythloda — APPROVED** [review]
❤️

**Comment by reinsch82** [comment]
@bot-gradle test and merge

**Comment by reinsch82** [comment]
@bot-gradle test and merge

**Pull request merged**